APIKit/APIKit.swift

```diff
@@ -72,47 +75,9 @@ public class API: NSObject, NSURLSessionDelegate, NSURLSessionDataDelegate {
     public class func responseBodyParser() -> ResponseBodyParser {
         return .JSON(readingOptions: nil)
     }
-    
-    public class func URLSessionConfiguration() -> NSURLSessionConfiguration {
-        return NSURLSessionConfiguration.defaultSessionConfiguration()
-    }
-    
-    public class func URLSessionDelegateQueue() -> NSOperationQueue? {
-        // nil indicates NSURLSession creates its own serial operation queue.
-        // see doc of NSURLSession.init(configuration:delegate:delegateQueue:) for more details.
-        return nil
-    }
-    
-    // prevent instantiation
-    override private init() {
```

Comment:
Oh, the initializer has been removed in ad0b537c2db70cc8e841881dd48b6590235e2b75! I have misunderstood. Anyway, initializer of `API` is not accessible from out of APIKit because it is now pure Swift class :relieved: